crypto/Kconfig

```diff
@@ -0,0 +1,10 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+# This file is autogenerated, do not edit.
+#
+
+menu "Cryptography Support"
+source "/home/bashton/nuttx/wrk/apps/crypto/mbedtls/Kconfig"
```

Comment:
A reminder to change the path here.

Reply:
Added the gitignore since this is auto-generated.

Reply:
Ah, right.